src/cljdoc/doc_tree.clj

```diff
@@ -152,16 +165,18 @@
   "For various purposes it is useful to know the path to a given document
   in it's doc tree. This function adds a field in [:attrs :slug-path] that
   contains the slug of the current document preceded by all slugs of documents
-  above it in the tree.
+  above it in the tree. Entries without a slug are ignored.
```

Comment:
```suggestion
  above it in the tree. External URLs that don't have slugs are ignored.
```
Both fine, just felt like that tells the person reading this why there might be entries without slugs.